Assets/Scripts/Common/Constants.cs

```diff
@@ -7,8 +7,17 @@ namespace Assets.Scripts.Common
 {
     public static class Constants
     {
-        public const float XZ_BLOCK_MOVE_SPEED = 1f;
-        public const float Y_BLOCK_MOVE_SPEED = 1f;
-        public const float BLOCK_OUT_OF_BOUNDS_Y = -2.0f;
+        public enum Coords : int
+        {
+            x = 0,
+            z = 1,
+            y = 2
+        }
+
+        //if you have a problem with these not being constant, stiff
+        public static float[] BLOCK_PLACE_SPEED = { 1f, 1f, 1f };
+        public static float[] BLOCK_BOUNDS_UPPER = { 20f, 10f, 500f };
```

Comment:
i think making a simple class with X, Y, Z props would make more sense than storing these values in an array

Reply:
or something like `Constants.Block.Bounds.UPPER_X` or `Constants.Block.Bounds.Upper.X`, if it's easier

Reply:
This isn't how I wanted to do it but stupid lil floats make it harder, I'm not opposed to class, let me think about it